webapp/channels/src/components/channel_header_dropdown/menu_items/close_channel/close_channel.tsx

```diff
@@ -7,25 +7,31 @@ import Menu from 'components/widgets/menu/menu';
 
 import {localizeMessage} from 'utils/utils';
 
-type Props = {
+interface CloseChannelProps {
     isArchived: boolean;
     actions: {
         goToLastViewedChannel: () => void;
     };
 }
 
-export default class CloseChannel extends React.PureComponent<Props> {
-    private handleClose = () => {
-        this.props.actions.goToLastViewedChannel();
+const CloseChannel = ({
+    isArchived,
+    actions,
+}: CloseChannelProps): JSX.Element => {
+    const handleClose = () => {
+        actions.goToLastViewedChannel();
     };
 
-    render() {
-        return (
-            <Menu.ItemAction
-                show={this.props.isArchived}
-                onClick={this.handleClose}
-                text={localizeMessage('center_panel.archived.closeChannel', 'Close Channel')}
-            />
-        );
-    }
-}
+    return (
+        <Menu.ItemAction
+            show={isArchived}
+            onClick={handleClose}
+            text={localizeMessage(
```

Comment:
`localizeMessage` is a function we want to get rid of. In this case you should use `intl.formatMessage`, and for that, you should do `const intl = useIntl();` in the component.